docs/design-systems/index.md

```diff
@@ -1,9 +1,100 @@
+---
+title: Design System | bluefox.email documentation
+description: Discover how the bluefox.email design system helps you build consistent, reusable email components with variables, blocks, and templates. Ensure uniformity across projects while enabling customization at the project level.
+head:
+  - - meta
+    - name: description
+      content: Discover how the bluefox.email design system helps you build consistent, reusable email components with variables, blocks, and templates. Ensure uniformity across projects while enabling customization at the project level.
+  - - meta
+    - property: og:title
+      content: Design System | bluefox.email documentation
+  - - meta
+    - property: og:description
+      content: Discover how the bluefox.email design system helps you build consistent, reusable email components with variables, blocks, and templates. Ensure uniformity across projects while enabling customization at the project level.
+  - - meta
+    - property: og:image
+      content: https://bluefox.email/assets/docs-share.png
+  - - meta
+    - property: og:url
+      content: https://bluefox.email/docs/projects
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: twitter:card
+      content: summary_large_image
+  - - meta
+    - property: twitter:title
+      content: Design System | bluefox.email documentation
+  - - meta
+    - property: twitter:description
+      content: Discover how the bluefox.email design system helps you build consistent, reusable email components with variables, blocks, and templates. Ensure uniformity across projects while enabling customization at the project level.
+  - - meta
+    - property: twitter:image
+      content: https://bluefox.email/assets/docs-share.png
+---
+
 # Design systems
 
-//TODO
+The design system in our email product is a comprehensive framework that helps users build, customize, and manage email designs efficiently. It allows for the creation of reusable components, ensuring consistency across all email communications. With this system, you can design emails from start to finish without needing to write any code, using our drag-and-drop editor.
+
+It provides a streamlined approach to email design. Users can create reusable blocks and templates, define variables for global customization, and associate design systems with specific projects. This makes the process of managing email styles and content consistent and efficient across all campaigns.
+
+## Design System Principles
+The design system follows Atomic Design, breaking emails into reusable components for consistency and scalability.
+
+- Atoms – Basic elements like colors and text styles.
+- Molecules – Functional groups, such as buttons with icons.
+- Organisms – Larger sections like headers or footers.
+- Templates – Define email layouts.
+- Pages – Final email designs for campaigns.
```

Comment:
I would remove the pages part, it's kinda hard to apply it here.

Reply:
basically the "sent out emails" are the pages here if we think about it that way.